packages/node/src/lib/novu.ts

```diff
@@ -4,13 +4,15 @@ import { EventEmitter } from 'events';
 import { Changes } from './changes/changes';
 import { INovuConfiguration } from './novu.interface';
 import { Events } from './events/events';
+import { NotificationGroups } from './notificationGroups/notificationGroups';
```

Comment:
A really small note, could you please rename the files and folder to be:
`notification-groups/notification-groups.ts` using a `-` instead of camelCasing? to match our file and folder structure better

Reply:
Done! @scopsy . PTAL

Reply:
cc @scopsy Please check and merge if you get a chance. Thanks1 :)